docs/troubleshooting.md

```diff
@@ -1,3 +1,49 @@
 # Troubleshooting Guide
 
-Content to be added.
+## Manual cleanup
+
+During Abort/Finalize LCA performs a cleanup. This includes:
+
+- Remove the old state root
+- Cleanup precaching resources
+- Delete OADP backups CRs
+- Remove IBU files from the file system
+
+If LCA fails to perform any of the above steps it transitions to Abort/Finalize failed state.
+The condition message and log shows which steps have failed.
+User should perform manual cleanup and add `lca.openshift.io/manualCleanupDone` annotation to IBU. After observing this annotation LCA will retry the cleanup and if successful, IBU transitions to `Idle`.
+
+### Stateroot cleanup
+
+During `Abort` LCA cleanups the new stateroot and during `Finalize` LCA cleanups the old stateroot. User should inspect the log to determine why this step has failed since failure in this step indicates serious issue in node.
```

Comment:
```suggestion
During `Abort` LCA cleanups the new stateroot and during `Finalize` LCA cleanups the old stateroot. User should inspect the log to determine why this step has failed since failure in this step indicates a serious issue in the node.
```